src/styles/blocks/Search-box-small.css

```diff
@@ -0,0 +1,44 @@
+.small_search-bar {
+  margin: 20px 8px;
+
+}
+
+.small_search-bar__input {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  background: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+
+  background-size: 11px;
+  background-image: url(../../images/Search.svg);
+  box-sizing: border-box;
+  background-repeat: no-repeat;
+  background-position: 12px 14px;
```

Comment:
Some properties are the same for both inputs. Consider putting them under the common selector